src/dataframerow/dataframerow.jl

```diff
@@ -1,4 +1,8 @@
-# Container for a DataFrame row
+"""
+    DataFrameRow
```

Comment:
Should mention the type parameter. Same below.

Reply:
Fixed. I have also added supertype informatin where applicable (here I think it is especially relevant as many users think that `DataFrameRow` a subtype of `AbstractDataFrame`).